packages/vanilla/src/complex/TableArrayControl.tsx

```diff
@@ -83,10 +87,10 @@ class TableArrayControl extends RendererComponent<TableProps, void> {
     const buttonClass = getStyleAsClassName('array.table.button');
     const controlClass = [getStyleAsClassName('array.table'),
       convertToValidClassName(controlElement.scope)].join(' ');
-    const createControlElement = (key: string): ControlElement => ({
+    const createControlElement = (key = ''): ControlElement => ({
```

Comment:
shouldn't this be rather optional, as in the case of an object array you want to have the key and in the case of primitives you never want to have this key